go.mod

```diff
@@ -28,3 +28,6 @@ require (
 	google.golang.org/grpc v1.42.0 // indirect
 	gopkg.in/check.v1 v1.0.0-20201130134442-10cb98267c6c // indirect
 )
+
+// This forces transitive deps to use a version of `image-spec` which addresses https://github.com/advisories/GHSA-77vh-xpmg-72qh
+replace github.com/opencontainers/image-spec => github.com/opencontainers/image-spec v1.0.2-0.20211117181255-693428a734f5
```

Comment:
Out of curiosity, what transitive dependency picks this up?

Since the "fix" isn't even included in the dependency -- clients and registries need to adopt the guidance in the updated spec -- I'm pretty opposed to this relatively heavyweight solution. This is a weird change to make just to silence dependabot alerts, which can just be marked as ignored.

Would it help to have an issue where we canonically describe the dependabot alert, what it means, what it _doesn't_ mean (i.e., that you're vulnerable), and call it solved?

Reply:
```
$ go mod graph | grep github.com/opencontainers/image-spec
github.com/google/go-containerregistry github.com/opencontainers/image-spec@v1.0.2-0.20211117181255-693428a734f5
github.com/containerd/containerd@v1.5.8 github.com/opencontainers/image-spec@v1.0.1
github.com/containerd/imgcrypt@v1.1.1 github.com/opencontainers/image-spec@v1.0.1
github.com/containerd/containerd@v1.5.0-beta.3 github.com/opencontainers/image-spec@v1.0.1
github.com/containerd/containerd@v1.5.0-rc.0 github.com/opencontainers/image-spec@v1.0.1
github.com/containers/ocicrypt@v1.1.1 github.com/opencontainers/image-spec@v1.0.1
github.com/containerd/aufs@v0.0.0-20201003224125-76a6863f2989 github.com/opencontainers/image-spec@v1.0.1
github.com/containerd/imgcrypt@v1.0.4-0.20210301171431-0ae5c75f59ba github.com/opencontainers/image-spec@v1.0.1
github.com/containerd/nri@v0.0.0-20201007170849-eb1350a75164 github.com/opencontainers/image-spec@v1.0.1
github.com/containerd/zfs@v0.0.0-20210301145711-11e8f1707f62 github.com/opencontainers/image-spec@v1.0.1
github.com/containerd/containerd@v1.5.0-beta.4 github.com/opencontainers/image-spec@v1.0.1
github.com/containerd/imgcrypt@v1.1.1-0.20210312161619-7ed62a527887 github.com/opencontainers/image-spec@v1.0.1
github.com/containerd/containerd@v1.5.0-beta.1 github.com/opencontainers/image-spec@v1.0.1
github.com/containers/ocicrypt@v1.1.0 github.com/opencontainers/image-spec@v1.0.1
github.com/containerd/aufs@v0.0.0-20200908144142-dab0cbea06f4 github.com/opencontainers/image-spec@v1.0.1
github.com/containerd/imgcrypt@v1.0.1 github.com/opencontainers/image-spec@v1.0.1
github.com/containerd/zfs@v0.0.0-20200918131355-0a33824f23a2 github.com/opencontainers/image-spec@v1.0.1
github.com/docker/distribution@v0.0.0-20190905152932-14b96e55d84c github.com/opencontainers/image-spec@v1.0.0
github.com/containers/ocicrypt@v1.0.1 github.com/opencontainers/image-spec@v1.0.1
```

> Since the "fix" isn't even included in the dependency -- clients and registries need to adopt the guidance in the updated spec -- I'm pretty opposed to this relatively heavyweight solution. This is a weird change to make just to silence dependabot alerts, which can just be marked as ignored.

agreed :/

> Would it help to have an issue where we canonically describe the dependabot alert, what it means, what it doesn't mean (i.e., that you're vulnerable), and call it solved?

sgtm